examples/index.html

```diff
@@ -21,14 +21,7 @@
     </style>
   </head>
   <body>
-    <div class="Dashboard-canvas js-dashboard">
-      <div class="Dashboard-mapWrapper Dashboard-mapWrapper--margin">
-        <div class="Map-canvas">
-          <div class="Map" id="map"></div>
-        </div>
-      </div>
-      <div class="Dashboard-belowMap js-dashboard-belowMap"></div>
-    </div>
+    <div class="Dashboard-canvas" id="dashboard"></div>
```

Comment:
How would this look if the `.Dashboard-canvas` class is added here? Should we ensure our views add this class?